y2023/ChargedUp/src/main/java/com/gos/chargedup/RobotContainer.java

```diff
@@ -123,6 +133,39 @@ private void createTestCommands() {
         createClawTestCommands();
         createArmTestCommands();
         createPivotTestCommands();
+        createSwerveTestCommands();
+
+    }
+
+    private void createSwerveTestCommands() {
+        ShuffleboardTab tab = Shuffleboard.getTab("SwerveDriveTestCommands");
+
+        tab.add("45deg Swerve Module 0", m_swerveDrive.commandSetModuleState(0, 45, 3));
+        tab.add("45deg Swerve Module 1", m_swerveDrive.commandSetModuleState(1, 45, 3));
+        tab.add("45deg Swerve Module 2", m_swerveDrive.commandSetModuleState(2, 45, 3));
+        tab.add("45deg Swerve Module 3", m_swerveDrive.commandSetModuleState(3, 45, 3));
```

Comment:
Can we translate these to for loops? I feel like we might want a whole swath of these when we go to test

```suggestion
        for (int i = 0; i < 4; ++i) {
            tab.add("45deg Swerve Module " + i, m_swerveDrive.commandSetModuleState(i, 45, 3));
        }
```